source/ui/SGM/SGMXASScanConfigurationView.cpp

```diff
@@ -11,6 +13,15 @@ SGMXASScanConfigurationView::SGMXASScanConfigurationView(SGMXASScanConfiguration
 	configuration_ = configuration;
 	detectors_ = 0;
 
+	gratingAngleControl_ = new SGMGratingAngleControl(this);
```

Comment:
Could the gratingAngleControl_ or the gratingTranslationStepControl_ be used anywhere else? If so, perhaps they could be passed into the view as parameters?